### PR TITLE
Update tool versions March 2019

### PR DESCRIPTION
**Pull request by akito19**
We updated the tools' version.

### CODE REVIEW

File: docs/tools/python/flake8.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ hide_title: true
 
 | Supported Version | Language | Web Site |
 | ----------------- | -------- | -------- |
-| 3.7.5 | Python 2.7.13, 3.6.2 | [http://flake8.pycqa.org/en/latest/](http://flake8.pycqa.org/en/latest/) |
+| 3.7.7 | Python 2.7.16, 3.7.2 | [http://flake8.pycqa.org/en/latest/](http://flake8.pycqa.org/en/latest/) |
```
Comment (akito19) [inline docs/tools/python/flake8.md]: Python versions were also updated: https://github.com/sider/devon_rex_python/pull/9 .

Comment (ybiquitous) [reply docs/tools/python/flake8.md]: Is `2.7.13` a mistake?

File: docs/tools/ruby/rubocop.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ hide_title: true
 
 | Supported Version | Language | Web Site |
 | ----------------- | -------- | -------- |
-| >= 0.35.0 (default to 0.64.0) | Ruby 2.5.1 | [https://github.com/rubocop-hq/rubocop](https://github.com/rubocop-hq/rubocop) |
+| >= 0.35.0 (default to 0.65.0) | Ruby 2.5.1 | [https://github.com/rubocop-hq/rubocop](https://github.com/rubocop-hq/rubocop) |
```
Comment (ybiquitous) [inline docs/tools/ruby/rubocop.md]: Is this supported minimum version `0.35.0` correct?

File: docs/tools/python/flake8.md (hunk 1 above)
Comment (akito19) [reply docs/tools/python/flake8.md]: No, `2.7.13` has already updated. https://github.com/sider/devon_rex_python/pull/8

Comment (ybiquitous) [reply docs/tools/python/flake8.md]: OK, thanks.

File: docs/tools/ruby/rubocop.md (hunk 2 above)
Comment (akito19) [reply docs/tools/ruby/rubocop.md]: Yes, you will confirm it in checking runner_rubocop.

Comment (ybiquitous) [reply docs/tools/ruby/rubocop.md]: Oh, I misunderstand with `RUBOCOP_VERSIONS` file. Sorry. 🙏

**Pull request merged**